charts/assertoor/Chart.yaml

```diff
@@ -6,7 +6,7 @@ home: https://github.com/ethpandaops/assertoor
 sources:
   - https://github.com/ethpandaops/assertoor
 type: application
-version: 0.0.4
+version: 0.0.5
```

Comment:
might wanna bump a major version for this.

Reply:
bumped to v1.0.0 :)